ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -765,20 +708,18 @@ export default class ConfirmTransactionBase extends Component {
         action={functionType}
         title={title}
         titleComponent={this.renderTitleComponent()}
-        subtitle={subtitle}
```

Comment:
I think subtitle can be removed from `ConfirmPageContainer` and down through the component tree, as was done with `summaryComponent`

Reply:
Good catch! Removed in 3b7176a8be50554766385e920dbe7dea3c5eb6d5